### PR TITLE
Add hs-mannheim.de

### PR DESCRIPTION
**Pull request by DavidLiedke**

### CODE REVIEW

File: src/chrome/content/rules/hs-mannheim.de.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		www.fsw.hs-mannheim.de
+		www.fotovoltaik.hs-mannheim.de
+	
+	Time Out:
+		www.gestaltung.hs-mannheim.de
+-->		
+<ruleset name="Hochschule Mannheim (partial)">
+
+	<!-- hs-mannheim.de -->
+	<target host="www.hs-mannheim.de"/>
```
Comment (J0WI) [inline src/chrome/content/rules/hs-mannheim.de.xml]: `www` is already covered by the wildcard, but the wildcard would also break all non-working hosts listed above.

**Comment by DavidLiedke** [comment]
@J0WI

**Comment by J0WI** [comment]
Can you add a comment to document the behavior that the non www version of the listed targets are serving a mismatching cert?

**Comment by DavidLiedke** [comment]
@J0WI 
I add a comment for the cert mismatch.

**Comment by J0WI** [comment]
LGTM, thanks!

**Pull request merged**